dags/services/AWSService.py

```diff
@@ -0,0 +1,40 @@
+import boto3
```

Comment:
This and `NextflowTowerService.py` will soon hold classes with all of the attributes and methods one would need to create a new DAG connecting to AWS resources or either of our Nextflow Tower instances. I created a ticket for this work in the "Migration of Airflow" Epic. The overall goal is to make DAGs more modular and make DAGs simple and readable by removing repetitive code from them.

Reply:
I think I agree with the "should" point. For now, I will continue with my plan to create these separate services this sprint and then they could be moved to sagetasks from there.